docs/upgrade.md

```diff
@@ -5,6 +5,12 @@ description: Guide to update between major Powertools versions
 
 <!-- markdownlint-disable MD043 -->
 
+## End of support v1
+
+On March 31, 2023, AWS Lambda Powertools for Python version 1 will reach end of support. After that, AWS Lambda Powertools for Python version 1 will no longer receive updates or releases. If you are still using v1, we encourage you to read our upgrade guide and update to the latest version.
+
+Given our commitment to all of our customers using AWS Lambda Powertools for Python, we won't remove old package versions from [Pypi](https://pypi.org/project/aws-lambda-powertools/) and will also leave old tags active, so you can read the documentation for v1 version.
```

Comment:
You can improve this paragraph by moving to a positive tone and by using active voice.

**Example**

> We will keep [PyPi](https://pypi.org/project/aws-lambda-powertools/) v1 releases and documentation 1.x versions to prevent any disruption.

Reply:
Thank you for this feedback, @heitorlessa ! It really makes a lot more sense to use active voice.